packages/core/examples/editableTextExample.tsx

```diff
@@ -78,15 +61,20 @@ export class EditableTextExample extends BaseExample<IEditableTextExampleState>
         return [
             [
                 <IntentSelect intent={this.state.intent} key="intent" onChange={this.handleIntentChange} />,
-                <label className={Classes.LABEL} key="maxlength">
-                    Max length
-                    <input
-                        className={Classes.INPUT}
+                <div className={Classes.FORM_GROUP} key="maxlength">
+                    <label className={Classes.LABEL} htmlFor={INPUT_ID}>Max length</label>
+                    <NumericInput
+                        id={INPUT_ID}
+                        className={Classes.FORM_CONTENT}
+                        min={0}
+                        max={300}
+                        onValueChange={this.handleMaxLengthChange}
                         placeholder="Unlimited"
-                        onChange={this.handleMaxLengthChange}
-                        value={this.state.maxLength !== undefined ? this.state.maxLength.toString() : ""}
+                        value={this.state.maxLength || ""}
                     />
-                </label>,
+                </div>,
+            ],
+            [
```

Comment:
I'd love to add a custom style for this example column to make it triple-wide. Looks a little goofy right now:

![image](https://cloud.githubusercontent.com/assets/443450/25864527/9d4a3788-34a4-11e7-8fc5-f0400b5cc91b.png)

This is what it could look like:

![image](https://cloud.githubusercontent.com/assets/443450/25864559/bdbd7e08-34a4-11e7-9e37-08150f32d70d.png)